.github/workflows/dispatch-build-bottle.yml

```
@@ -0,0 +1,127 @@
+name: Dispatch build bottle
+
+on:
+  workflow_dispatch:
+    inputs:
+      macos:
+        description: macOS version
+        required: true
+      formula:
+        description: Formula name
+        required: true
+      issue:
+        description: Issue number, where comment on failure would be posted
+        required: false
+      upload:
+        description: Whether to upload built bottles to Bintray or not
+        required: false
+
+jobs:
+  bottle:
+    runs-on: ${{github.event.inputs.macos}}
+    timeout-minutes: 4320
+    env:
+      HOMEBREW_GIT_NAME: BrewTestBot
+      HOMEBREW_GIT_EMAIL: 'homebrew-test-bot@lists.sfconservancy.org'
+      PATH: '/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin'
+      HOMEBREW_DEVELOPER: 1
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
+      GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED: 1
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - name: Update Homebrew
+        run: brew update-reset
+
+      - name: Set up Git repository
+        run: |
+          cd $(brew --repo ${{github.repository}})
+          git clean -ffdx
+          git fetch --prune --force origin master
+          git fetch --prune --force origin ${{github.sha}}
+          git checkout --force ${{github.sha}}
+          git log -1
+
+      - name: Run brew test-bot --only-cleanup-before
+        run: brew test-bot --only-cleanup-before
```

These automatically get "Run <command>" names
```suggestion
      - run: brew test-bot --only-cleanup-before
```